aiven/internal/service/wait.go

```diff
@@ -131,7 +131,7 @@ func WaitForServiceUpdate(ctx context.Context, d *schema.ResourceData, m interfa
 	return aux.(*aiven.Service), nil
 }
 
-func WaitAfterStaticIpsDissassociation(ctx context.Context, d *schema.ResourceData, m interface{}) error {
```

Comment:
Can you please add a comment to this function? It is getting hard to understand what each function does for the Static IP. For example, we already have two waiting functions: this one and `WaitBeforeStaticIpsDissassociation`.